### ISSUE TITLE
<Continue generative a11y feedback research> from <CAIA>

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#56294]
## a11y CAIA Asignee(s)
- Previously Ops Item with Terry & Eli (added to ticket)  #83204
- CAIA A11Ys (joint effort)
- Accessibility Community of Practice (Office Hours)

## What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 

Last year, we conducted initial [discovery research into how disabled Veterans provide accessibility feedback to the VA](https://docs.google.com/document/d/1L7EgD3wn0bKk3nnPQrFgKffopaz1awbfgZQ2qyVIQs4/edit?usp=sharing). This led to us publishing the new accessibility statement page, but we haven't heard much feedback since.
 
In a recent VAOS usability test with a disabled Veteran today, we received direct feedback on the accessibility statement page.

Some quotes from the session:

- [Reading the page] "'We're committed to providing access to our websites and digital applications...' [Laughs] It doesn't happen!" 
- "Yea that's all find and dandy what Section 508 is... but nothing ever gets accomplished."
- “I probably called and sent emails a dozen times [using this page]. Especially after I was trying to get employment on USA jobs... I have to have someone sighted to insert documentation there.”
- “That’s disability discrimination right there.”

Provided these quotes, and existing knowledge from our initial research with the Section 508 office and Tim Hornik (BVA), it would be valuable to continue generative research to better understand disabled Veteran's behaviors, feelings, expectations, and journeys providing a11y feedback (beyond the limiting scope of evaluative usability testing). This could then better inform future...
- participatory research and design strategy
- iterations and design of the a11y statement page
- collaboration with the Section 508 office
- feedback and design of alternative feedback mechanisms 

...in the hopes of creating more direct lines between our a11y work at VA.gov and Veterans directly, in line with what @artsymartha68 has evangelized for

Possible avenues forward worth discussion may include:
- Collaborating with VEO (and their recent work with the BVA)
- Outreach to local facilities (we have leads through @artsymartha68 in a recent call set up by Tim Hornik)
- Interviews with BVA representatives 
- Re-continuing to append questions related to a11y feedback to existing VA.gov AT tudies

## When does this work need to be done?
When time allows, preferably sooner than later

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by sara-amanda** [comment]
> @sara-amanda is this work still relevant?

Thank you for sending us this `Backlog` item to review, @strelichl 
- Yes, this was previously an A11Y Ops Item with Terry & Eli 
    - It has been connected and added the task list in this linked ticket: #83204
- I have also updated the assignees in this ticket's body and in the assignee field, along with the GHP fields. 

@NaomiPMC @artsymartha68 - perhaps we can add this to the list of tickets to discuss in our next A11Y/OCTO meeting, in addition to `New Intakes` - if time allows?

**Comment by artsymartha68** [comment]
Assigning to @cindymerrill 
This may be an old issue that can be closed, surpassed by all the AT research improvements she has underway.

**Comment by cindymerrill** [comment]
@artsymartha68 @coforma-jamie Not sure if we want to do anything with this, but having it on our slack canvas of research ideas seems reasonable.
